aws-datastore/src/main/java/com/amplifyframework/datastore/DataStoreConfiguration.java

```diff
@@ -44,21 +50,16 @@
     private final DataStoreConflictHandler conflictHandler;
     private final Integer syncMaxRecords;
     private final Integer syncPageSize;
-    private Long syncIntervalInMinutes;
-
-    private DataStoreConfiguration(
-            DataStoreErrorHandler errorHandler,
-            DataStoreConflictHandler conflictHandler,
-            Long syncIntervalInMinutes,
-            Integer syncMaxRecords,
-            Integer syncPageSize) {
-        this.errorHandler = errorHandler;
-        this.conflictHandler = conflictHandler;
-        this.syncMaxRecords = syncMaxRecords;
-        this.syncPageSize = syncPageSize;
-        if (syncIntervalInMinutes != null) {
-            this.syncIntervalInMinutes = syncIntervalInMinutes;
-        }
+    private final Map<Class<? extends Model>, DataStoreSyncExpression> syncExpressions;
```

Comment:
Can you make this a map from model name to sync expression, `Map<String, DataStoreSyncExpression>`? This would be more likely to work for Flutter, if/when that gets implemented in `amplify-flutter`.